paddle/fluid/ir_adaptor/translator/program_translator.cc

```diff
@@ -181,6 +181,10 @@ void ProgramTranslator::SetParameterFromSingleBlock(const BlockDesc& block) {
         need_set_parameter_op &= (parameter_visited_.count(var_name) == 0);
```

Comment:
```suggestion
        need_set_parameter_op &= (parameter_visited_.count(var_name) == 0);
        need_set_parameter_op &= (param_map_.count(var_name) != 0);
```
这样改应该就可以了，不需要下面再判断defining_op_result 和value是否为空，因为这两个不应该为nullptr，是的话应该报错